content/documentation/validations/index.adoc

```diff
@@ -687,10 +687,156 @@ include::/data/files/validation_examples/601.yaml[]
 https://github.com/kiali/kiali/blob/master/business/checkers/authorization/service_binding_checker.go[Validator source code, window="_blank"] +
 https://istio.io/docs/reference/config/authorization/istio.rbac.v1alpha1/#ServiceRoleBinding[Istio documentation, window="_blank"]
 
+[#sidecars]
+== Sidecars
+
+=== KIA1001 - No matching workload found for the sidecar selector in this namespace
+
+The Sidecar resources are used for configuring the sidecar proxies in the service mesh. This resource is aimed to target either specific workloads, all workloads in a namespace or all the sidecars of the service mesh. Each Sidecar resource living in a namespace (except the one living in the control plane namespace) effects only to the workloads in the same namespace.
+
+This validation refers to the scenario where there aren't workloads matching the labels specified into the sidecar's `workloadSelector` field. Therefore, all the settings defined within the sidecar resource won't be applied to any workload in the mesh.
+
+==== Resolution
+
+The validation will be gone when there is at least one workload matching the labels set in the `workloadSelector` field. The workload must be in the same namespace of the sidecar resource.
+
+==== Severity
+
+icon:exclamation-triangle[] Warning
+
+==== Example
+
+[source, yaml]
+----
+include::/data/files/validation_examples/901.yaml[]
+----
+
+==== See Also
+
+https://github.com/kiali/kiali/tree/master/business/checkers/sidecars/workload_selector_checker.go[Validator source code, window="_blank"] +
+https://istio.io/docs/reference/config/networking/sidecar/[Sidecar documentation, window="_blank"] +
+
+'''
+
+=== KIA1002 - More than one selector-less Sidecar in the same namespace
+
+The Sidecar resources are used for configuring the sidecar proxies in the service mesh. This resource is aimed to target either specific workloads, all workloads in a namespace or all the sidecars of the service mesh. Each Sidecar resource living in a namespace (except the one living in the control plane namespace) effects only to the workloads in the same namespace.
+
+Selector-less sidecars are those sidecars that doesn't have the `workloadSelector` field and thus they are pointing to all workloads running in the same namespace.
```

Comment:
"that doesn't" -> "that don't" I guess you are talking in plural in this sentence.